common/SQL/CDTCDPH_VACCINE/vaccine_hpi/vaccine_hpi.sql

```diff
@@ -3,18 +3,20 @@ select
     HPIQUARTILE,
     AGE16_POPULATION,
     FIRST_DOSE,
+    FIRST_DOSE_ONLY,
     COMPLETED_DOSE,
-    FIRST_DOSE+COMPLETED_DOSE "COMBINED_DOSES",
-    FIRST_DOSE/AGE16_POPULATION "FIRST_DOSE_RATIO",
+    FIRST_DOSE_ONLY+COMPLETED_DOSE "COMBINED_DOSES",
+    FIRST_DOSE_ONLY/AGE16_POPULATION "FIRST_DOSE_RATIO",
     COMPLETED_DOSE/AGE16_POPULATION "COMPLETED_DOSE_RATIO",
-    (FIRST_DOSE+COMPLETED_DOSE)/AGE16_POPULATION "COMBINED_DOSES_RATIO"
+    (FIRST_DOSE_ONLY+COMPLETED_DOSE)/AGE16_POPULATION "COMBINED_DOSES_RATIO"
```

Comment:
I think we don't want to change this line because the 2nd chart bar values should add up to the 10M doses delivered, not the 7M people given shots

Reply:
(worked out using combined doses as distinct vax_events